src/lib.rs

```diff
@@ -39,10 +39,22 @@ pub mod pallet {
         vec::Vec,
     };
 
+    /// Elo constant
+    /// See https://metinmediamath.wordpress.com/2013/11/27/how-to-calculate-the-elo-rating-including-example/
+    const K: f32 = 32_f32;
+
     pub type AssetIdOf<T> =
         <<T as Config>::Assets as Inspect<<T as frame_system::Config>::AccountId>>::AssetId;
     pub type BalanceOf<T> =
         <<T as Config>::Assets as Inspect<<T as frame_system::Config>::AccountId>>::Balance;
+    type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
+
+    pub struct DefaultElo;
+    impl Get<u16> for DefaultElo {
+        fn get() -> u16 {
+            1600
```

Comment:
Because all actual ELO systems I'm aware of start by that number. In any case it's just a convention, and any value would suffice. If your actual level is lower than that you'll start loosing games and quickly your ELO will decrease. Similarly if your actual strength is higher than 1600.

Reply:
It makes sense to have this number to start.